packages/server/start/routes.ts

```diff
@@ -171,6 +171,21 @@ Route.group(() => {
 
 /// //////// ////// ///
 
+Route.post(
+    '/me/playlists-visibility',
+    'UserSettingsController.updatePlaylistsVisibility',
+);
+
+Route.post(
+    '/me/relations-visibility',
+    'UserSettingsController.updateRelationsVisibility',
+);
+
+Route.post(
+    '/me/devices-visibility',
+    'UserSettingsController.updateDevicesVisibility',
+);
+
```

Comment:
interesting maybe should we refactor currently `user/my-profile-information` toward `me/my-profile-information` or either `me/profile-information` ?

Reply:
Yes, we should normalize our URLs 👍
Let's talk about that soon!